tests/resources/test_publish_errors.py

```diff
@@ -68,7 +72,7 @@ def test_nested_field_error(
     assert 400 == response.status_code
     expected = [
         {
-            "field": "metadata.creators.1.name",
+            "field": "metadata.creators.1.person_or_org.name",
```

Comment:
I've added a note to review frontend Formik fieldpaths in light of these changes in creators task:   https://github.com/inveniosoftware/react-invenio-deposit/issues/154#issuecomment-763700142